__init__.py

```diff
@@ -141,9 +143,13 @@ def execute(self, urls):
                 image1 = torch.cat((image1, image2), dim=0)
             return (image1,)
 
+ffmpeg_path = shutil.which("ffmpeg")
+
 class VideoCombine:
     """ 
     Batches images into WebP format and uploads them S3. 
+
+    Source: https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/blob/main/videohelpersuite/nodes.py#L165
```

Comment:
Looks like we'll need to update the license to GPL 3.0, not ideal. Doable if need be.

This code isn't quite working either in my local testing, so it might be better to try again from scratch. The structure here is also significantly lower than the structure there.

Reply:
Actually, looks like we can make an only "upload to s3" node once https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/pull/82 is merged.

Reply:
When this is merged, we can use their video combiner? Regarding structure, it's more lightweight because we're not dealing with audio files, metadata, or multiple file formats.

Reply:
Yeah, when that is merged, we can drop VideoCombine and just add an upload to s3 node.

Will update the license if they ask us to before they merge and we remove the derivative code.